doc/nrf/templates/library_template_README.rst

```diff
@@ -0,0 +1,143 @@
+.. _library_template:
```

Comment:
Do want to give guidance on how to name the reference tag?
For example, do we want all libraries to start with `lib_`, or other such recommendations?

Reply:
Many of the library docs have a reference tag starting with _lib_something, but not all of them.

Reply:
I added this as a suggestion, as there are already libraries which don't use the lib_ prefix